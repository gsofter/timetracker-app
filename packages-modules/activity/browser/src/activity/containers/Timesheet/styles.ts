import { Property, Properties } from 'csstype';

export const styleSheet: { [key: string]: (obj) => Properties } = {
  timesheet: ({ theme }) => {
    const { primaryColor } = theme;
    return {
      position: 'relative',
      background: '#fff',
      padding: '24px',
      '& .ant-timeline-item-head-blue': {
        borderColor: primaryColor ? primaryColor : '#1b8efa',
      },
      '& .card-outer': {
        display: 'flex',
        justifyContent: 'space-between',
        flexWrap: 'wrap',
      },
      '& .no-activity': {
        flexWrap: 'wrap',
        background: '#ddd',
        width: '170px',
        height: '50px',
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
        margin: '100px 10px',
        borderRadius: '2px',
        '@media (max-width: 668px)': {
          width: '120px !important',
        },
        '@media (min-width: 992px)': {
          width: '140px !important',
        },
        '& p': {
          padding: '0px',
          margin: '0px',
          color: '#181818',
        },
      },
      '& .activity-card': {
        width: '170px',
        padding: '30px 0px',
        textAlign: 'center',
        margin: '0px 10px',
        '@media (max-width: 668px)': {
          width: '120px !important',
        },
        '@media (min-width: 992px)': {
          width: '140px !important',
        },
        '& h4': {
          padding: '5px 10px',
          boxSizing: 'border-box',
          borderRadius: '50px',
          background: '#ddd',
          margin: '0px',
        },
        '& p': {
          fontSize: '13px',
          padding: '10px 0px',
          margin: '0px',
        },
      },
      '& .ant-card-body': {
        padding: '0px',
      },
      '& .activity-details': {
        '& img': {
          width: '100%',
        },
        '& .activity-body': {
          padding: '15px 10px',
          borderLeft: '1px solid #ddd',
          borderRight: '1px solid #ddd',
          borderBottom: 'solid 1px #ddd',
          borderRadius: '0px 0px 3px 3px',
        },
        '& .time-horizontal': {
          display: 'flex',
          justifyContent: 'space-between',
          marginBottom: '5px',
          fontSize: '11px',
          '& .icon-value': {
            display: 'inharit',
            fontSize: '11px',
            '& svg': {
              marginRight: '5px',
            },
          },
        },
        '& .progress-card': {
          textAlign: 'center',
        },
      },
      '& .ant-progress-bg': {
        height: '4px !important',
      },
      '@media (max-width: 768px)': {
        padding: '10px !important',
      },
    };
  },
};
