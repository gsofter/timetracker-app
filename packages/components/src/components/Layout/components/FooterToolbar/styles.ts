import { IThemeState, getTokenColorsScopeSettings } from '../../../Themes/redux';
import { GlobalThemeSettings } from '../default-styles';


export const styleSheet = {
    footerToolbarStyle: ({ theme, prefixCls }: { theme: IThemeState, prefixCls: string }) => {
        const { antPrefix, componentBackground, borderColorSplit, shadow1Up } = GlobalThemeSettings;

        // less variables
        const proFooterBarPrefixCls = `${antPrefix}-pro-footer-bar`;


        return {
            [`& .${proFooterBarPrefixCls}`]: {
                position: 'fixed',
                right: '0',
                bottom: '0',
                'z-index': '99',
                display: 'flex',
                width: '100%',
                'align-items': 'center',
                padding: '0 24px',
                'line-height': '44px',
                background: `${componentBackground}`,
                'border-top': `1px solid ${borderColorSplit}`,
                'box-shadow': `${shadow1Up}`,
                '&-left': {
                    flex: '1',
                },

                '&-right': {
                    '> *': {
                        'margin-right': '8px',
                        '&:last-child': {
                            margin: '0',
                        }
                    }
                }
            },
        };
    },
}