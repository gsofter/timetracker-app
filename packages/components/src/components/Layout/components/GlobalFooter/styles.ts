// import { IThemeState, getTokenColorsScopeSettings } from '../../../Themes/redux';
import { ITheme } from '../../../Themes/interfaces';

import { GlobalThemeSettings } from '../default-styles';
import { getTokenColorsScopeSettings } from '../../../Themes/utils';


export const styleSheet = {
  globalFooter: ({ theme, prefixCls }: { theme: ITheme, prefixCls: string }) => {
    const { antPrefix, fontSizeBase } = GlobalThemeSettings;

    // less variables
    const proGlobalFooterPRefixCls = `${antPrefix}-pro-global-footer`;
    const textColorSecondary = getTokenColorsScopeSettings(theme, 'text.secondary').foreground;
    const textColor = getTokenColorsScopeSettings(theme, 'text').foreground;

    return {
      [`& .${proGlobalFooterPRefixCls}`]: {
        margin: '48px 0 24px 0',
        padding: '0 16px',
        'text-align': 'center',

        '&-links': {
          'margin-bottom': ' 8px',

          '& a': {
            color: `${textColorSecondary}`,
            transition: 'all 0.3s',

            '&:not(:last-child)': {
              'margin-right': '40px',
            },

            '&:hover': {
              color: `${textColor}`,
            },
          },
        },

        '&-copyright': {
          color: `${textColorSecondary}`,
          'font-size': `${fontSizeBase}`,
        },
      },
    };
  },
};
