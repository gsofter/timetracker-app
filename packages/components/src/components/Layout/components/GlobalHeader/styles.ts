import { Property, Properties } from 'csstype';

export const styleSheet: { [key: string]: (obj) => Properties } = {
    globalHeaderStyle: ({ theme, primaryColor, layout }) => ({
        display: 'inherit',
        '& .ant-pro-global-header': {
            position: 'relative',
            display: 'flex',
            alignItems: 'center',
            height: '100%',
            padding: '0 16px',
            background: '#fff',
            boxShadow: '0 1px 4px rgba(0,21,41,0.08)',
        },
        '& .ant-pro-global-header > *': {
            height: '100%',
        },
        '& .ant-pro-global-header-collapsed-button': {
            display: 'flex',
            alignItems: 'center',
            marginLeft: '16px',
            fontSize: '20px',
        },
        '& .ant-pro-global-header-layout-mix': {
            backgroundColor: '#001529',
        },
        '& .ant-pro-global-header-layout-mix .ant-pro-global-header-logo h1': {
            color: '#fff',
        },
        '& .ant-pro-global-header-layout-mix .anticon': {
            color: '#fff',
        },
        '& .ant-pro-global-header-logo': {
            position: 'relative',
            overflow: 'hidden',
        },
        '& .ant-pro-global-header-logo a': {
            display: 'flex',
            alignItems: 'center',
            height: '100%',
        },
        '& .ant-pro-global-header-logo a img': {
            height: '28px',
        },
        '& .ant-pro-global-header-logo a h1': {
            height: '32px',
            margin: '0 0 0 12px',
            color: '#1890ff',
            fontWeight: '600',
            fontSize: '18px',
            lineHeight: '32px',
        },
        '& .ant-pro-global-header-menu .anticon': {
            marginRight: '8px',
        },
        '& .ant-pro-global-header-menu .ant-dropdown-menu-item': {
            minWidth: '160px',
        },
        '& .ant-pro-global-header .dark': {
            height: '48px',
        },
        '& .ant-pro-global-header .dark .action': {
            color: 'rgba(255,255,255,0.85)',
        },
        '& .ant-pro-global-header .dark .action > i': {
            color: 'rgba(255,255,255,0.85)',
        },
        '& .ant-pro-global-header .dark .action:hover, .ant-pro-global-header .dark .action.opened': {
            background: '#1890ff',
        },
        '& .ant-pro-global-header .dark .action .ant-badge': {
            color: 'rgba(255,255,255,0.85)',
        },

    }),
};

