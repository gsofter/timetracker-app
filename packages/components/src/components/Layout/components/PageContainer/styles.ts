import { IThemeSettings } from '../../../Themes/interfaces';
import { GlobalThemeSettings } from '../default-styles';


export const styleSheet = {
	pageContainer: ({ theme }: { theme: IThemeSettings }) => {
		const { antPrefix, screenLg, screenMd, screenSm, screenXl } = GlobalThemeSettings;

		// less variables
		const proLayoutPageContainer = `${antPrefix}-pro-page-container`;


		return {
			[`& .${proLayoutPageContainer}-children-content`]: {
				margin: '24px 24px 0',
			},
			[`& .${proLayoutPageContainer}`]: {

				'&-warp': {
					'background-color': `${GlobalThemeSettings.componentBackground}`,
					[`& .${antPrefix}-tabs-nav`]: {
						margin: '0',
					},
				},
				'&-ghost': {
					[`& .${proLayoutPageContainer}-warp`]: {
						'background-color': 'transparent',
					},

					[`& .${proLayoutPageContainer}-children-content`]: {
						'margin-top': '0',
					}
				}
			},
			[`& .${proLayoutPageContainer}-main`]: {
				[`& .${proLayoutPageContainer}-detail`]: {
					display: 'flex',
				},

				[`& .${proLayoutPageContainer}-row`]: {
					display: 'flex',
					width: '100%',
				},

				[`& .${proLayoutPageContainer}-title-content`]: {
					'margin-bottom': '16px',
				},

				[`& .${proLayoutPageContainer}-title`]: {
					flex: 'auto',

				},
				[`& .${proLayoutPageContainer}-content`]: { // merge it into one
					flex: 'auto',
					'margin-bottom': '16px',
				},

				[`& .${proLayoutPageContainer}-extraContent`]: { // merge it into one
					flex: '0 1 auto',
					'min-width': '242px',
					'margin-left': '88px',
					'text-align': 'right',
				},
				[`& .${proLayoutPageContainer}-main`]: {
					flex: '0 1 auto',

					width: '100%',
				},
				[`& .${proLayoutPageContainer}-logo`]: {
					'margin-bottom': '16px',
				},
			},
			[`@media screen and (max-width: ${screenXl})`]: {
				[`& .${proLayoutPageContainer}-main`]: {
					[`& .${proLayoutPageContainer}-extraContent`]: {
						'margin-left': '44px',
					}
				}
			},
			
			[`@media screen and (max-width: ${screenLg})`]: {
				[`& .${proLayoutPageContainer}-main`]: {
					[`& .${proLayoutPageContainer}-extraContent`]: {
						'margin-left': '20px',
					}
				}
			},
			
			[`@media screen and (max-width: ${screenMd})`]: {
				[`& .${proLayoutPageContainer}-main`]: {
					[`& .${proLayoutPageContainer}-row`]: {
						display: 'block',
					},
			
					[`& .${proLayoutPageContainer}-action`]: {
						'margin-left': '0',
						'text-align': 'left',
					},
					[`& .${proLayoutPageContainer}-extraContent`]: {
						'margin-left': '0',
						'text-align': 'left',
					}
				}
			},
			
			[`@media screen and (max-width: ${screenSm})`]: {
				[`& .${proLayoutPageContainer}-detail`]: {
					display: 'block',
				},
				[`& .${proLayoutPageContainer}-extraContent`]: {
					'margin-left': '0',
				},
			}
		};
	},
};
