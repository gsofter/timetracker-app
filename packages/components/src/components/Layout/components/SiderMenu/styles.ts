import { Property, Properties } from 'csstype'

export const styleSheet: { [key: string]: (obj) => Properties } = {
    siderMenuStyles: ({ theme, primaryColor }) => ({
        display: 'inherit',
        '& .ant-pro-sider': {
            position: 'relative',
            borderRight: '0',
            transition: 'background-color 0.3s',
        },
        '& .ant-layout-sider': {
            height: '100%',
        },
        '& .ant-pro-sider.ant-menu-vertical .ant-menu-item:not(:last-child), .ant-pro-sider.ant-menu-vertical-left .ant-menu-item:not(:last-child), .ant-pro-sider.ant-menu-vertical-right .ant-menu-item:not(:last-child), .ant-pro-sider.ant-menu-inline .ant-menu-item:not(:last-child)': {
            marginBottom: '4px',
        },
        '& .ant-drawer-close': {
            top: '-13px',
            right: '-17px',
            color: theme === 'light' ? primaryColor : '#fff',
        },
        '& .ant-drawer-content': {
            backgroundColor: theme === 'light' ? '#000' : '#fff',
        },
        '& .ant-pro-sider.ant-layout-sider-light .ant-menu-item a, .ant-menu-submenu-selected, .ant-menu-submenu-title:hover, .ant-menu-submenu-title i:hover ': {
            color: primaryColor ? primaryColor : '#1890ff',
        },
        '& .ant-menu-submenu-inline > .ant-menu-submenu-title:hover .ant-menu-submenu-arrow::before, .ant-menu-submenu-inline > .ant-menu-submenu-title:hover .ant-menu-submenu-arrow::after': {
            background: `linear-gradient(to right, ${primaryColor ? primaryColor : '#1890ff'}, ${primaryColor ? primaryColor : '#1890ff'})`,
        },
        '& .ant-pro-sider.ant-layout-sider-light, .ant-pro-sider.ant-layout-sider-light .ant-menu-item:not(.ant-menu-item-selected) a:hover, .ant-menu-item:hover': {
            color: primaryColor ? primaryColor : '#1890ff',
        },
        '& .ant-menu-item-selected a, .ant-menu-item-selected a:hover, .ant-menu:not(.ant-menu-horizontal) .ant-menu-item-selected a:hover': {
            color: '#fff',
        },
        '& .ant-menu:not(.ant-menu-horizontal) .ant-menu-item-selected': {
            background: primaryColor ? primaryColor : '#1890ff',
        },
        '& .removeBoxShadow ul': {
            boxShadow: 'none',
        },
        '& .ant-menu-inline .ant-menu-item::after': {
            borderRight: 'none',
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-item .anticon + span, .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-item .anticon + span, .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-submenu > .ant-menu-submenu-title .anticon + span, .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-submenu > .ant-menu-submenu-title .anticon + span': {
            display: 'none',
        },
        '& .ant-pro-sider ul.ant-menu-sub li.ant-menu-item .ant-pro-menu-item, .ant-pro-sider ul.ant-menu-sub li.ant-menu-submenu .ant-pro-menu-item': {
            paddingLeft: '8px',
        },
        '& .ant-pro-sider-logo': {
            position: 'relative',
            display: 'flex',
            alignItems: 'center',
            padding: '16px 16px',
            lineHeight: '32px',
            cursor: 'pointer',
        },
        '& .cu .ant-pro-sider-menu': {
            boxShadow: 'none',
        },
        '& .ant-pro-sider-logo > a': {
            display: 'flex',
            alignItems: 'center',
            justifyContent: 'center',
            minHeight: '32px',
        },
        '& .ant-pro-sider-logo img': {
            display: 'inline-block',
            height: '32px',
            verticalAlign: 'middle',
            transition: 'height 0.2s',
        },
        '& .ant-pro-sider-logo h1': {
            display: 'inline-block',
            height: '32px',
            margin: '0 0 0 12px',
            color: theme === 'light' ? primaryColor : '#fff',
            fontWeight: '600',
            fontSize: '18px',
            lineHeight: '32px',
            verticalAlign: 'middle',
            animation: 'fade-in',
            animationDuration: '0.2s',
        },
        '& .ant-pro-sider-extra': {
            marginBottom: '16px',
            padding: '0 16px',
        },
        '& .ant-pro-sider-extra-no-logo': {
            marginTop: '16px',
        },
        '& .ant-pro-sider-menu': {
            position: 'relative',
            zIndex: '10',
            minHeight: '100%',
            boxShadow: '2px 0 6px rgba(0,21,41,0.35)',
        },
        '& .ant-pro-sider .ant-layout-sider-children': {
            display: 'flex',
            flexDirection: 'column',
            height: '100%',
        },
        '& .ant-pro-sider .ant-layout-sider-children ::-webkit-scrollbar': {
            width: '6px',
            height: '6px',
        },
        '& .ant-pro-sider .ant-layout-sider-children ::-webkit-scrollbar-track': {
            background: 'rgba(255,255,255,0.15)',
            borderRadius: '3px',
            webkitBoxShadow: 'inset 0 0 5px rgba(37,37,37,0.05)',
        },
        '& .ant-pro-sider .ant-layout-sider-children ::-webkit-scrollbar-thumb': {
            background: 'rgba(255,255,255,0.2)',
            borderRadius: '3px',
            webkitBoxShadow: 'inset 0 0 5px rgba(255,255,255,0.05)',
        },
        '& .ant-pro-sider.ant-layout-sider-collapsed .ant-menu-inline-collapsed': {
            width: '48px',
        },
        '& .ant-pro-sider.ant-layout-sider-collapsed .ant-pro-sider-logo': {
            padding: '16px 8px',
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-item .sider-menu-item-img + span, .ant-pro-sider .-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-item .sider-menu-item-img + span, .ant-pro-sider .ant-menu-inline-collapsed > .ant-menu-submenu > .ant-menu-submenu-title .sider-menu-item-img + span': {
            display: 'inline-block',
            maxWidth: 0,
            opacity: 0,
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed .ant-menu-item': {
            width: '48px',
            padding: '0 16px !important',
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed .ant-pro-menu-item-title': {
            display: 'none',
        },
        '& .ant-pro-sider .ant-menu-inline-collapsed .ant-menu-submenu-title': {
            width: '48px',
            padding: '0 16px !important',
        },
        '& .ant-pro-sider .ant-menu-item .anticon, .ant-pro-sider .ant-menu-submenu-title .anticon': {
            transition: 'none',
        },
        '& .ant-pro-sider-fixed': {
            position: 'fixed',
            top: '0',
            left: '0',
            zIndex: '99',
            height: '100%',
            overflow: 'auto',
            overflowx: 'hidden',
            boxShadow: '2px 0 8px 0 rgba(29,35,41,0.05)',
        },
        '& .ant-pro-sider-fixed > .ant-menu-root :not(.ant-pro-sider-link-menu)': {
            height: 'calc(100vh - 64px)',
            overflowY: 'auto',
        },
        '& .ant-pro-sider .ant-menu-inline .ant-menu-item, .ant-pro-sider .ant-menu-inline .ant-menu-submenu-title': {
            width: '100%',
        },
        '& .ant-pro-sider-light': {
            boxShadow: '2px 0 8px 0 rgba(29,35,41,0.05)',
        },
        '& .ant-pro-sider-light .ant-layout-sider-children ::-webkit-scrollbar-track': {
            background: 'rgba(0,0,0,0.06)',
            borderRadius: '3px',
            webkitBoxShadow: 'inset 0 0 5px rgba(0,21,41,0.05)',
        },
        '& .ant-pro-sider-light .ant-layout-sider-children ::-webkit-scrollbar-thumb': {
            background: 'rgba(0,0,0,0.12)',
            borderRadius: '3px',
            webkitBoxShadow: 'inset 0 0 5px rgba(0,21,41,0.05)',
        },
        '& .ant-pro-sider-light .ant-pro-sider-logo h1': {
            color: theme === 'light' ? primaryColor : '#fff',
        },
        '& .ant-pro-sider-light .ant-menu-light': {
            borderRightColor: 'transparent',
        },
        '& .ant-pro-sider-icon': {
            width: '14px',
            verticalAlign: 'baseline',
        },
        '& .ant-pro-sider-links': {
            width: '100%',
        },
        '&.ant-pro-sider-links ul.-menu-root': {
            height: 'auto',
        },
        '& .ant-pro-sider-collapsed-button': {
            borderTop: 'rgba(0,0,0,0.25)',
        },
        '& .ant-divider': {
            color: theme === 'light' ? 'rgba(0, 0, 0, 0.50)' : 'rgba(255, 255, 255, 0.65)',
            borderTopColor: theme === 'light' ? 'rgba(0, 0, 0, 0.50)' : 'rgba(255, 255, 255, 0.65)',
        },
        '& .ant-pro-sider-collapsed-button .anticon': {
            fontSize: '16px',
        },
        '& .ant-pro-sider .top-nav-menu li.ant-menu-item': {
            height: '100%',
            lineHeight: 1,
        },
        '& .ant-pro-sider .drawer .drawer-content': {
            //  background: ;
        },
        '& .ant-pro-sider .ant-menu-item .sider-menu-item-img + span, .ant-pro-sider .ant-menu-submenu-title .sider-menu-item-img + span': {
            opacity: 1,
            transition: 'opacity 0.3s , width 0.3s',
        },
        '& .ant-menu-inline-collapsed > .ant-menu-item, .ant-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-item, .ant-menu-inline-collapsed > .ant-menu-item-group > .ant-menu-item-group-list > .ant-menu-submenu > .ant-menu-submenu-title, .ant-menu-inline-collapsed > .ant-menu-submenu > .ant-menu-submenu-title': {
            padding: '0 24px',
        },
        '& .ant-pro-sider-menu.mobile-base-menu .ant-menu-submenu-title': {
            height: 'auto',
            lineHeight: '25px',
            whiteSpace: 'normal',
        },
        '& .ant-pro-sider-menu.mobile-base-menu .ant-menu-submenu-title .ant-pro-menu-item': {
            display: 'flex',
        },
        '& .ant-pro-sider-menu.mobile-base-menu .ant-menu-submenu-title .ant-pro-menu-item .anticon': {
            paddingTop: '5px',
        },
        '@keyframes fade-in': {
            '0%': {
                display: 'none',
                opacity: 0,
            },
            '99%': {
                display: 'none',
                opacity: 0,
            },
            '100%': {
                display: 'block',
                opacity: 1,
            },
        },
    }),
}
