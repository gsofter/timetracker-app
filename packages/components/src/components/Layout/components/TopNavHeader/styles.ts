import { getTokenColorsScopeSettings } from '../../../Themes/redux/utils';
import { IThemeState } from '../../../Themes/redux';
import { GlobalThemeSettings, LocalThemeSettings } from '../default-styles';


export const styleSheet = {
    topNavHeader: ({ theme, prefixCls }: { theme: IThemeState, prefixCls: string }) => {
        const { defaultSettings: { colors: { primaryColor }, type: themeName } } = theme;
        const { componentBackground, btnPrimaryColor, antPrefix } = GlobalThemeSettings;

        const headerColor = getTokenColorsScopeSettings(theme, 'header');
        // less variables
        const topNavHeaderPrefixCls = `${antPrefix}-pro-top-nav-header`;
        const headingColor = headerColor.foreground;
        return {
            [`& .${topNavHeaderPrefixCls}`]: {
                position: 'relative',
                width: '100%',
                height: '100%',
                'box-shadow': '0 1px 4px 0 rgba(0, 21, 41, 0.12)',
                transition: 'background 0.3s, width 0.2s',

                [`& .${antPrefix}-menu`]: {
                    background: 'transparent',
                },
                '&.light': {
                    'background-color': `${componentBackground}`,
                    [`& .${topNavHeaderPrefixCls}-logo`]: {
                        '& h1': {
                            color: `${headingColor}`,
                        },
                    },
                    '& .anticon': {
                        color: 'inherit',
                    }
                },
                '&-main': {
                    display: 'flex',
                    height: '100%',
                    'padding-left': '16px',
                    '&-left': {
                        display: 'flex',
                        'min-width': '192px',
                    },
                },

                '& .anticon': {
                    color: `${btnPrimaryColor}`,
                },
                '&-logo': {
                    position: 'relative',
                    'min-width': '165px',
                    height: '100%',
                    overflow: 'hidden',

                    '& img': {
                        display: 'inline-block',
                        height: '32px',
                        'vertical-align': 'middle',
                    },

                    '& h1': {
                        display: 'inline-block',
                        margin: '0 0 0 12px',
                        color: `${btnPrimaryColor}`,
                        'font-weight': '400',
                        'font-size': '16px',
                        'vertical-align': 'top',
                    }
                }
            },
        };
    },
};
