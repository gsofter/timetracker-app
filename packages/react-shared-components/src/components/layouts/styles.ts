export const styleSheet = {
  siderLayoutStyles: () => {
    return {
      '& .ant-pro-sider-extra': {
        overflow: 'hidden auto',
      },
      '& .ant-pro-sider-extra::-webkit-scrollbar': {
        width: '4px',
        borderRadius: '2px'
      },
      '& .ant-pro-sider-extra::-webkit-scrollbar-track': {
        background: '#253747',
      },
      '& .ant-pro-sider-extra::-webkit-scrollbar-thumb': {
        background: '#888',
      },
      '& .ant-pro-sider-extra::-webkit-scrollbar-thumb:hover': {
        background: '#555'
      },
      '& .ant-pro-sider-menu': {
        minHeight: 'auto !important',
      },
    };
  },
};