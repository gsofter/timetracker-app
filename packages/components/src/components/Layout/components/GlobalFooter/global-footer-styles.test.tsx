import React from 'react';
import * as renderer from 'react-test-renderer';
import { createRenderer } from 'fela';
import { Provider, createComponent } from 'react-fela';

import { styleSheet } from './styles';
import 'jest';

const styles = styleSheet();
const Box = createComponent(styles.footerLayout);

describe('Box', () => {
    it('should render component', () => {
      const felaRenderer = createRenderer();
      console.log('----test---', felaRenderer)
      const component = renderer.create(
        <Provider renderer={felaRenderer}>
          <Box>hello</Box>
        </Provider>
      )
  
      expect(component.toJSON()).toMatchSnapshot()
    })
  })
