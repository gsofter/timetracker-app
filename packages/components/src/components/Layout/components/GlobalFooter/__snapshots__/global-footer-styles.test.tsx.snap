// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test styles should return a Component 1`] = `
".a .ant-pro-global-footer {
  margin: 48px 0 24px 0;
}
.b .ant-pro-global-footer {
  padding: 0 16px;
}
.c .ant-pro-global-footer {
  text-align: center;
}
.d .ant-pro-global-footer-links {
  margin-bottom: 8px;
}
.e .ant-pro-global-footer-links a {
  color: #8c8c8c;
}
.f .ant-pro-global-footer-links a {
  transition: all 0.3s;
}
.g .ant-pro-global-footer-links a:not(:last-child) {
  margin-right: 40px;
}
.h .ant-pro-global-footer-links a:hover {
  color: #595959;
}
.i .ant-pro-global-footer-copyright {
  color: #8c8c8c;
}
.j .ant-pro-global-footer-copyright {
  font-size: 14px;
}


<div className=ant-pro a b c d e f g h i j />;
"
`;
